bin/cmds/app/view.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const colors = require('colors');
+const { Log } = require('../../..');
+const { App } = require('../../..');
+
+exports.desc = 'View your app in the Homey Apps Store';
+exports.handler = async yargs => {
+  const appPath = yargs.path || process.cwd();
```

Comment:
I know this is done like this in other places but its not necessary. `yargs.path` is always defined...